fecgov/fec-cms
# Add new homepage icon links and banner

## Summary
- Add new icon links to homepage. Use `.grid__item--with-icon` classes with some additions for  homepage-specific design.
- Make this section of homepage editable in wagtail 
- Add new homepage banner

- [ ]  After merge, do steps 4 and 5 in How-to-test, in production.

@JonellaCulmer : In terms of making the line breaks match the mockup: This is the closest I could get. There are some  imperfect techniques for controlling the line-breaks (like wrapping in spans or adding non-breaking spaces (`&nbsp;`)) but these don't really work with this layout, break at responsive sized and require cluttering the element with  extraneous, non-semantic tags. One good thing with the current layout is that the bottom of each of the three items, lines up symmetrically. But we can go over this further.

- Resolves #3544
- Resolves #3585

## Impacted areas of the application
	modified:   fec/static/scss/layout/_grid.scss
	modified:   home/templates/home/home_page.html

	new file: fec/static/img/i-complex--legal--gray-lightest.svg
	new file: fec/static/img/i-complex--regreport--gray-lightest.svg
        new file: fec/static/img/map-chart-icon.svg

## Screenshots
<img width="1326" alt="Screen Shot 2020-03-08 at 10 05 40 PM" src="https://user-images.githubusercontent.com/5572856/76176963-0295b180-6189-11ea-8d85-d9b62b91271d.png">

## How to test
1)  checkout `feature/3544-add-new-homepage-icon-links`
2)  `npm run build-sass`
3)   Open homepage for editing in Wagtail.
4)  Add the  html below into an `html-block` in `body`
5)  Preview or publish page to see new icon links and banner image
6)  Test links go to right place
```
<ul class="grid grid--3-wide grid--flex u-padding--bottom">
  <li class="grid__item grid__item--with-icon icon--large grid__icon--white text-centered">
    <img src="/static/img/map-chart-icon.svg" alt="icon representing Presidential candidate map">
    <h2 class="u-margin--top"><a href="/data/candidates/president/presidential-map/">Presidential candidate map</a></h2>
    <p>Search presidential data such as money raised, money spent, cash on hand and debt.</p>
  </li>
  <li class="grid__item grid__item--with-icon icon--large grid__icon--white text-centered">
    <img src="/static/img/i-complex--regreport--gray-lightest.svg" alt="icon representing help for candidates and committees">
    <h2 class="u-margin--top"><a href="/help-candidates-and-committees/">Help for candidates and committees</a></h2>
    <p>Help for individuals and groups who are active in federal elections.</p>
  </li>
  <li class="grid__item grid__item--with-icon icon--large grid__icon--white text-centered">
    <img src="/static/img/i-complex--legal--gray-lightest.svg" alt="icon representing legal resources search">
    <h2 class="u-margin--top"><a href="/legal-resources/">Research legal resources</a></h2>
    <p>Explore relevant statutes, regulations, Commission actions and court cases.</p>
  </li>
</ul>

```
- [ ]  After merge, do steps 4 and 5 in How-to-test, in production.

____
